container-storage-setup.sh

```diff
@@ -34,8 +34,8 @@ STORAGE_IN_FILE="/etc/sysconfig/docker-storage-setup"
 STORAGE_OUT_FILE="/etc/sysconfig/docker-storage"
 STORAGE_DRIVERS="devicemapper overlay overlay2"
 
-PIPE1=/run/dss-fifo1
-PIPE2=/run/dss-fifo2
+PIPE1=/run/css-fifo1
+PIPE2=/run/css-fifo2
```

Comment:
You mean create a new pair of pipes for each running instance? I think child needs to know in advance the name of pipe to wait on.